migrate/users/minisdk.go

```diff
@@ -0,0 +1,80 @@
+package users
+
+import (
+	"context"
+	"encoding/csv"
+	"fmt"
+
+	mf13users "github.com/mainflux/mainflux/users"
+	mf13postgres "github.com/mainflux/mainflux/users/postgres"
+	"github.com/mainflux/migrations/internal/util"
+)
+
+var (
+	offset            = uint64(0)
+	limit             = uint64(100)
+	retrievUsersOps   = "retrieving users"
+	writeUsersOps     = "writing users to csv file"
+	retrieveErrString = "error %v occured at offset: %d and total: %d during %s"
+)
+
+// MFRetrieveUsers retrieves existing users from the database
+func MFRetrieveUsers(ctx context.Context, db mf13postgres.Database) (mf13users.UserPage, error) {
+	usersPage, err := RetrieveAllUsers(ctx, db, mf13users.PageMetadata{Offset: offset, Limit: limit})
+	if err != nil {
+		return mf13users.UserPage{}, fmt.Errorf(retrieveErrString, err, offset, limit, retrievUsersOps)
+	}
+	o := limit
+	limit = util.UpdateLimit(usersPage.Total)
+	for o < usersPage.Total {
```

Comment:
The idea of fetching everything from the DB to the RAM. That's the point of pagination. This should be more like a pipe where we stream (channel) the batch, and publish it to the next phase to process. Bear in mind that we'll need to remember "last saved" so in case something goes wrong we don't need to start over.